sass/_chatbox.scss

```diff
@@ -255,6 +255,17 @@
                 width: 100%;
                 resize: none;
             }
+            .chat-textarea-send-button {
+                height: $chat-textarea-height - 35px;
+            }
+            .send-button {
+                position: absolute;
+                bottom: 5px;
+                right: 10px;
```

Comment:
If we change them to ``em`` then need to change others to em as well. Also these are just position values so I think keeping them in pixels will not be a problem. Please correct me if I am wrong somewhere.